Portfolio Project/style.css

```diff
@@ -1,3 +1,3 @@
 #logo {
-    color: blue;
-}
\ No newline at end of file
+  color: blue;
```

Comment:
- [ ] Good work on this project 👍 . I noticed that the CSS stylesheet you linked in your HTML does not contain the styling for the HTML view. Kindly link the correct stylesheet to your HTML file 😄.

This is what your UI looks like 👇 
![image](https://user-images.githubusercontent.com/63473584/178082536-133807f8-7cf3-4a8e-8307-2983f86393c4.png)